.github/workflows/sync-addon-metadata-translations.yml

```diff
@@ -0,0 +1,46 @@
+name: Sync addon metadata translations
```

Comment:
Can you sync this file with https://github.com/kodi-game/game.libretro.2048/blob/master/.github/workflows/sync-addon-metadata-translations.yml?

Reply:
Have it updated and should now commit direct, instead of pull requests.